### PR TITLE
OpenEditorAction.py: Add notepadqq to known editors

### PR DESCRIPTION
**Pull request by guilherme-p**
Couldn't find any documentation regarding command line arguments.
Tried several commands to enter into a specific line/column but none
worked. 'notepadqq {filename}' works.

Closes https://github.com/coala/coala/issues/5244

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by guilherme-p** [comment]
@palash25 Was having trouble squashing commits so I created a new PR

**Comment by palash25** [comment]
use this for future reference http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by guilherme-p** [comment]
I used rebase, but I squashed commits in the wrong order, which messed up my commit history beyond repair.

File: coalib/results/result_actions/OpenEditorAction.py [hunk 1]
```diff
@@ -82,6 +82,10 @@
         'args': '--new',
         'gui': True
     },
+    'notepadqq': {
+        'file_arg_template': '{filename}',
+        'gui': True
```
Comment (lalitmee) [inline coalib/results/result_actions/OpenEditorAction.py]: Hey @guilherme-p , you didn't include any `args` property. Are you sure that **notepadqq** will not have any argument?

**Comment by guilherme-p** [comment]
I’m not sure at all. Like I said in the commit, I couldn’t find any arguments, but if someone more experienced with nqq could chime in I’d appreciate it.

File: coalib/results/result_actions/OpenEditorAction.py (hunk 1 above)
Comment (jayvdb) [reply coalib/results/result_actions/OpenEditorAction.py]: Waiting for feedback from @palash25 who created the issue

Comment (newbazz) [reply coalib/results/result_actions/OpenEditorAction.py]: ping @palash25

**Comment by jayvdb** [comment]
ping @palash25, they have provided a useful reply on that issue, and I also replied with a general reply.  Could you take a look and decide whether we should be using the 'new window' mode.

**Comment by palash25** [comment]
@jayvdb I read the issue thread on notepadqq. Whenever I use notepadqq I use multiple tabs in the same window, as a coala user I wouldn't want multiple windows everytime I run coala.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5258/commits/67003a1985caaf75544b051c9f98b42ec2a6aa67

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**